### ISSUE TITLE
Test for tcp address connections

### ISSUE THREAD

**Issue by acrisci** [altdesktop/python-dbus-next#60]
#57 

What is the use case for tcp address connections? Are people using dbus config or some kind of network forwarding?

@M-Bab

**Comment by M-Bab** [comment]
I guess mainly people are using dbus config. We have a successful config here on an embedded device with anonymous auth but this is only for testing. Obviously the combination of opening up dbus to the network combined with anonymous authentication is somehow dangerous.

**Comment by acrisci** [comment]
Yes that is very dangerous because it means anyone can call dbus methods on your system and some of the methods are extremely powerful.

Right now the tests just use network forwarding. These should be expanded to test with a direct tcp connection using a config. If you have an example config to try, you can post it here.
